### PR TITLE
options loading from profile if has profile loaded

### PR DESCRIPTION
**Pull request by ssreerama**
This PR Adds
1. The ability to the deployment options to load form the profiles xml, if the profile is loaded. If not, it will load the default options form the DacFx.

### CODE REVIEW

File: extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export class PublishDatabaseDialog {
 	private serverName: string | undefined;
 	protected optionsButton: azdataType.ButtonComponent | undefined;
 	private publishOptionsDialog: PublishOptionsDialog | undefined;
+	public fileUris: vscode.Uri[] | undefined;
```
Comment (kisantia) [inline extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts:62]: can this have a more descriptive name? When would there be multiple fileUris?

**Comment by Charles-Gagnon** [comment]
@ssreerama Is this still needed?

File: extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts (hunk 1 above)
Comment (ssreerama) [reply extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts:61]: This variable is for getting the profile options when profile exists, but we are not at that decision to how we handle the profile, I'm removing this logic, and default options will be the profile options if profile is loaded else deployoptions but reset always be default deployoptions as is till we decide it.

File: extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts [hunk 2]
```diff
@@ -836,6 +836,9 @@ export class PublishDatabaseDialog {
 					this.targetDatabaseName = result.databaseName;
 				}
 
+				// Set profiles options to the deployment options
+				this.setDeploymentOptions(result.options);
```
Comment (llali) [inline extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts:840]: add more comment explaining this is coming from profile

**Review by llali — APPROVED** [review]

**Review by Benjin — APPROVED** [review]

**Pull request merged**